pages/docs/v3/core-apis/saving-calls.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Persisting Plugin Calls
+description: How to save plugin calls in Capacitor
+---
+
+# Saving Plugin Calls
+
+In most cases, a plugin method will get invoked to perform a task and can finish immediately. But more complicated situations can require additional consideration.
+
+## Overview
+
+There are two reasons you might need a plugin call (`CAPPluginCall` on iOS or `PluginCall` on Android) to persist outside of the method in your plugin:
+
+1. To perform an asynchronous action, such as requesting permission from the system to access a protected API.
```

Comment:
With the new permission APIs this is no longer needed, right @carlpoole?

Reply:
I was trying to think of an async API that's involves a common task across platforms. But it's a good point that permissions are not the best choice with the recent changes to Android. Can you think of a different API to use as an example?

Reply:
network requests?